tools/gn

```diff
@@ -291,8 +295,12 @@ def to_gn_args(args):
   # No cross-compilation on Windows (for now). Use host toolchain that
   # matches the bit-width of the target architecture.
   if sys.platform.startswith(('cygwin', 'win')) and args.target_os != 'win':
-    gn_args['host_cpu'] = cpu_for_target_arch(gn_args['target_cpu'])
-    gn_args['target_cpu'] = cpu_for_target_arch(gn_args['target_cpu'])
+    cpu = cpu_for_target_arch(gn_args['target_cpu'])
+    # consider native arm64 build
```

Comment:
Nit: This should be a sentence, with a capital letter and a period.

Reply:
Done